etc/kubernetes/manifests/base/apps/cos-fleetshard-sync/crds/managedconnectors.cos.bf2.org-v1.yml

```diff
@@ -19,7 +19,7 @@ spec:
     - jsonPath: .spec.connectorId
       name: CONNECTOR_ID
       type: string
-    - jsonPath: .spec.deployment.connectorTypeId
+    - jsonPath: .status.deployment.connectorTypeId
```

Comment:
I believe this is changing to the wrong version?

Reply:
I must rebuild to get the resource sanitization reset it